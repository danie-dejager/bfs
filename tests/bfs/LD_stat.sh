bfs_diff -LD stat links
