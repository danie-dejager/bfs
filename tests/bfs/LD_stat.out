links
links/broken
links/deeply
links/deeply/nested
links/deeply/nested/broken
links/deeply/nested/dir
links/deeply/nested/file
links/deeply/nested/link
links/file
links/hardlink
links/notdir
links/skip
links/skip/broken
links/skip/dir
links/skip/file
links/skip/link
links/symlink
